src/web/src/app/space/invitation-list/component.html

```diff
@@ -0,0 +1,89 @@
+<ng-container *ngIf="total > 0">
+	<div class="heading">
+		<h2 i18n="@@SPACES.DETAIL.INVITATIONS.HEADING">Приглашения</h2>
+		<button *ngIf="currentSpace" mat-icon-button (click)="onAdd(currentSpace)"><mat-icon>add</mat-icon></button>
+	</div>
+	<div>
+		<!-- form for inviting user -->
+		<form *ngIf="!isFormHidden" class="row we-invited pad-bot-30" [formGroup]="form" (ngSubmit)="onSubmit()">
+			<div *ngIf="!currentSpace">
+				<span>{{formSpaceName}}</span>
+			</div>
+			<div class="fill-width">
+				<mat-form-field appearance="fill">
+					<input matInput type="text" formControlName="login" [matAutocomplete]="auto" required spaceUserExists (input)="onLoginChange($event.target.value)"/>
```

Comment:
Пользователь существует если его выбрали из выпадающего списка.
```suggestion
					<input matInput type="text" formControlName="login" [matAutocomplete]="auto" required (input)="onLoginChange($event.target.value)"/>
```

Reply:
Готово (убрал директиву, условия переписал).